fastlane/Fastfile

```diff
@@ -115,7 +115,6 @@ lane :publish_release do |options|
   UI.user_error!("You need to pass the version of the release") unless version.length > 0
 
   # Create XCFrameworks
-  xcversion(version: "13.0")
```

Comment:
Do not remove this, because the other step is pointing to ~> 13.0, which will mean 13.0.1 if there is any fix released.

It will also make it future proof

Reply:
Otherwise we need to change the `before_all` to point to an specific version, which could also be an option